examples/fastspeech/train_fastspeech.py

```diff
@@ -324,6 +331,10 @@ def main():
         )
         fastspeech._build()
         fastspeech.summary()
+        
+        if len(args.pretrained) > 1:
+            print("Loading pretrained parameters...")
+            fastspeech.load_weights(args.pretrained,by_name=True,skip_mismatch=True)
```

Comment:
should use space after each parameters, 

`fastspeech.load_weights(args.pretrained, by_name=True, skip_mismatch=True)`. Do the same for other files.